Duplicati/Library/Main/Volumes/VolumeWriterBase.cs

```diff
@@ -52,9 +52,6 @@ protected VolumeWriterBase(Options options, DateTime timestamp)
             else
                 m_localfile = new Library.Utility.TempFile();
 
-            // TODO(danstahr): This is a hack! Figure out why the file is being disposed of prematurely.
-            m_localfile.Protected = true;
-

```

Comment:
This removes a workaround that was introduced in revision 31ac40083ab85926b656e87f937b90dd02e69a5f in pull request #2637.  From the comments in that pull request, it appears that there were issues when running tests in Travis with temporary files being removed prematurely.  It's unclear if that is still an issue (it appears that the underlying cause was unknown), but hopefully it has been resolved.

This prevented the temporary files created during a compact operation from being removed.